src/SFML/Network/TcpSocket.cpp

```diff
@@ -68,7 +68,7 @@ unsigned short TcpSocket::getLocalPort() const
         priv::SocketImpl::AddrLength size = sizeof(address);
         if (getsockname(getHandle(), reinterpret_cast<sockaddr*>(&address), &size) != -1)
         {
-            return ntohs(address.sin_port);
+            return swapEndian(address.sin_port);
```

Comment:
I don't think we necessarily need to stop using `ntohs` and related functions. The motivation for this PR is to fix issues with how `sf::Packet` handles floats so I'd rather not touch other classes.

Reply:
Reverted all except `Packet.cpp`